images/compilers/install-bazel.sh

```diff
@@ -22,7 +22,7 @@ for bazel_version in "${bazel_versions[@]}" ; do
   # instead of using installer script, download binaries directly, as installer script doesn't append version suffix,
   # so for multiple versions to be usable files will need to moved around, which would be more tedious
   long_binary_name="bazel-${bazel_version}-linux-x86_64"
-  curl --fail --show-error --silent --location "https://releases.bazel.build/${bazel_version}/release/${long_binary_name}" --output "/usr/local/bin/${long_binary_name}"
+  curl -k --fail --show-error --silent --location "https://releases.bazel.build/${bazel_version}/release/${long_binary_name}" --output "/usr/local/bin/${long_binary_name}"
```

Comment:
Why is this now required? The commit description doesn't say.

Reply:
> The certificate for releases.bazel.build has expired

Sorry I was a little bit too brief, basically the cert for releases.bazel.build is just expired today.


<img width="346" alt="image (1)" src="https://user-images.githubusercontent.com/9019229/169039899-a9b79506-682c-4593-a68e-6c69e62b7259.png">

Reply:
Seems resolved now.

Reply:
thanks, let me revert this insecure flag :100: